q-gate.html

```diff
@@ -108,6 +111,19 @@
         <label for="node-input-resetCmd"><i class="fa fa-eject"></i> Reset</label>
         <input type="text" id="node-input-resetCmd" placeholder="reset" style="width:20%">
     </div>
+<!-- peekCmd & dropCmd --> 
```

Comment:
I will probably want to use different icons for `peekCmd` and `dropCmd`, if I can find suitable ones. This probably done best after merging your changes.

Reply:
Changed three icons and moved statusCmd in the edit dialog.